apps/cowswap-frontend/src/modules/application/containers/TradeWidgetLinks/index.tsx

```diff
@@ -23,7 +23,7 @@ const MENU_ITEMS: MenuItemConfig[] = [
   { route: Routes.LIMIT_ORDER, label: 'Limit' },
   {
     route: Routes.ADVANCED_ORDERS,
-    label: 'Advanced',
+    label: 'TWAP',
```

Comment:
Makes sense but, what about all the materials already out there (and the URL too) saying `advanced`?
Won't that confuse more users?

Reply:
I'm not sure.
But I hope it will at least reduce the current level of confusion :)

Reply:
I mean, people will sort it out :) 

Is not like, I'm looking for TWAP, but the guide says I have to go to Advanced. I don't see advanced, i just see TWAP (not sure how to continue :P )